pelican/readers.py

```diff
@@ -268,7 +268,11 @@ def _parse_metadata(self, meta):
                 formatted_values = "\n".join(value)
                 # reset the markdown instance to clear any state
                 self._md.reset()
+                # process field as markdown, but *don't* extract md metadata
+                preprocessors = self._md.preprocessors
+                self._md.preprocessors.pop('meta', None)
                 formatted = self._md.convert(formatted_values)
+                self._md.preprocessors = preprocessors
```

Comment:
Actually, how you handle it is fine. What I was trying to say that, two lines below don't do anything:

```python
preprocessors = self._md.preprocessors
#...
self._md.preprocessors = preprocessors
```

Just remove them :).